docs/data/base/components/click-away-listener/click-away-listener.md

```diff
@@ -1,38 +1,59 @@
 ---
 product: base
-title: React ClickAwayListener component
+title: Detect click outside React component
 components: ClickAwayListener
 githubLabel: 'component: ClickAwayListener'
 packageName: '@mui/base'
 ---
 
 # Click-away listener
 
-<p class="description">The <code>ClickAwayListener</code> component detects when a click event happens outside of its child element.</p>
+<p class="description">The ClickAwayListener component detects when a click event happens outside of its child element.</p>
 
-`ClickAwayListener` is a utility component that listens for click events outside of its child.
+## Introduction
```

Comment:
Is the plan to add this `## Introduction` in the rest of the docs? It seems that in Material UI we have systematically skipped it (the introduction content was left to be described under the h1), it's not consistent.

Reply:
Yes, I think we should apply this format across the Material UI docs as well.